src/utils/clm_time_manager.F90

```diff
@@ -540,12 +546,6 @@ subroutine timemgr_restart( )
 
     ! Initialize ref date from restart info
 
-    ref_date = TimeSetymd( rst_ref_ymd, rst_ref_tod, "ref_date" )
-
-    ! Initialize clock 
-
-    call init_clock( start_date, ref_date, curr_date)
-
```

Comment:
At the end also remove

if (masterproc) call timemgr_print()

As it's already done in the timemgr_init step previously. No reason to repeat.

Reply:
Moving this as a thing to do in https://github.com/ESCOMP/CTSM/pull/2912